### PR TITLE
Add infernalrestraints xPath scene scrapper

### PR DESCRIPTION
**Pull request by HeavyDinosaur**
Add infernalrestraints scene scrapper

### CODE REVIEW

File: scrapers/infernalrestraints.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      - infernalrestraints.com/device
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //a[@class="articleTitleText"][1]/text()
+      Date:
+        selector: //span[@class="articlePostDateText"][1]/text()
+        postProcess:
+          - parseDate: "January 02, 2006"
```
Comment (bnkai) [inline scrapers/infernalrestraints.yml:14]: this `          - parseDate: "January 02, 2006"` isnt needed
`January 02, 2006` is for sites that use 0 padding for single digit days eg April 01, 2020
`January 2, 2006` parses sites with April 1, 2020 instead
Both parse April 11, 2020

File: scrapers/infernalrestraints.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+          - parseDate: "January 2, 2006"
+      Details:
+        selector: //td[@class="articleCopyText"]/text()
```
Comment (bnkai) [inline scrapers/infernalrestraints.yml:17]: the <br> 's in the details are already parsed so you shouldn't need to do all that
```yaml
      Details:
        selector: //td[@class="articleCopyText"]/text()
        concat: "\n\n"
```
should be enough

File: scrapers/infernalrestraints.yml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+          postProcess:
+            - replace:
+              - regex: "Tags:"
+                with:
+          split: ","
+      Studio:
+        Name:
+            fixed: Infernal Restraints
+      Image: //*[@id="my-video"]/@poster
+      URL: //meta[@name='og:url']/@content
```
Comment (bnkai) [inline scrapers/infernalrestraints.yml:38]: is the og:url meta present in any scenes ? couldn't find a sample with it.

File: scrapers/infernalrestraints.yml [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+      Performers:
+        Name: //a[@class="articleTitleText"][2]/text()
+      Tags:
+        Name:
+          selector: //td[contains(text(),"Tags:")]/text()
+          # To remove the Tags: text from the tags
+          postProcess:
+            - replace:
+              - regex: "Tags:"
+                with:
```
Comment (bnkai) [inline scrapers/infernalrestraints.yml:32]: The tags seem to have some extra spaces included  after the comma. It doesn't show up in the UI but is shows in the console log if you set it to debug. 
Adding an extra regex after the tags replace fixes that
```yaml
            - replace:
              - regex: "Tags:"
                with:
              - regex: ",\\s*([^,]+)"
                with: ",$1"
```

File: scrapers/infernalrestraints.yml (hunk 3 above)
Comment (HeavyDinosaur) [reply scrapers/infernalrestraints.yml:38]: forgot to remove, inherited from another scrapper.

File: scrapers/infernalrestraints.yml (hunk 1 above)
Comment (HeavyDinosaur) [reply scrapers/infernalrestraints.yml:14]: fixed

File: scrapers/infernalrestraints.yml (hunk 2 above)
Comment (HeavyDinosaur) [reply scrapers/infernalrestraints.yml:17]: fixed

File: scrapers/infernalrestraints.yml (hunk 4 above)
Comment (HeavyDinosaur) [reply scrapers/infernalrestraints.yml:32]: updated

**Pull request merged**